src/main/kotlin/com/cognifide/gradle/aem/internal/ProgressLogger.kt

```diff
@@ -1,14 +1,22 @@
 package com.cognifide.gradle.aem.internal
 
+import org.apache.commons.lang3.time.StopWatch
 import org.gradle.api.Project
 import org.gradle.api.logging.Logger
 import org.gradle.internal.logging.progress.ProgressLoggerFactory
+import java.util.concurrent.TimeUnit
 
-class ProgressLogger(val project: Project, header: String) {
+class ProgressLogger(val project: Project, val header: String) {
 
-    val logger: Logger = project.logger
+    private val logger: Logger = project.logger
 
-    val progressLogger = create(header)
+    private val progressLogger = create(header)
+
+    private var stopWatch = StopWatch()
+
+    var progressEach: (String) -> Unit = { message -> logger.debug("$header: $message") }
+
+    var progressWindow = TimeUnit.SECONDS.toMillis(1)
```

Comment:
every one second progress will be visible in logger at level `DEBUG` to be able to debug it when build fail on CI server where progressbar is not visible